content/sidebar/student_finance.md

```diff
@@ -0,0 +1,10 @@
+title: К финансовым вопросам
+slug: student_finance
```

Comment:
`slug` name should not contain `_`. `slug` should be all lower case, and match `[a-z0-9-]*`.